b2sdk/replication/setup.py

```diff
@@ -195,11 +191,9 @@ def setup_source(
             file_name_prefix=prefix,
         )
         new_replication_configuration = ReplicationConfiguration(
-            ReplicationSourceConfiguration(
-                source_application_key_id=source_key.id_,
-                rules=current_source_rules + [new_rr],
-            ),
-            destination_configuration,
+            source_key_id=source_key.id_,
+            rules=current_source_rules + [new_rr],
+            **destination_configuration,
```

Comment:
```suggestion
            **destination_configuration
```

Reply:
https://stackoverflow.com/a/17492103/1935381

Reply:
yes, but you are never going to add anything after `**kwargs`, so trailing comma is a general rule but `**kwargs` is an exception from that rule :)